client/src/components/Forms/PreparedComment.tsx

```diff
@@ -0,0 +1,43 @@
+import { Typography } from 'antd';
+import { FC, useEffect, useState } from 'react';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+
+export const markdownLabel = '{markdown}\n';
+
+type Props = {
+  text: string;
+};
+
+const PreparedComment: FC<Props> = ({ text }) => {
+  const [state, setState] = useState(text);
+  const [isMD, setIsMD] = useState(false);
+
+  useEffect(() => {
+    if (text.indexOf(markdownLabel) === 0) {
+      setIsMD(true);
+      setState(text.slice(markdownLabel.length));
+    }
+  }, []);
+
+  /**
+   * Для того, чтобы не повлиять на предыдущие сохраненные комментарии, для markdown синтаксиса используем специальную метку.
```

Comment:
please try use english as we had and might have non-Russian speaking contributors.